### DISCUSSION TITLE
Funding choices

### DISCUSSION THREAD

**Discussion by raresvanca** [Shin-NiL/Godot-Android-Admob-Plugin#117] [Q&A] [answered]
I'm planning on releasing a game and I need ads. AdMob told me that if I want fewer ads to be blocked in UE and UK I should use Funding Choices. So my question here is: How I use Funding Choices in Godot and with this plugin? Or are there any alternatives? Or can I just ask myself the user for consent and inform AdMob about that?

Edit: I just reread the documentation. Is "is_personalized" what I am looking for?

**Comment by Shin-NiL** [comment, marked as answer]
The `is_personalized` property will just enable or disabled the personalized ads. The management of user consent is responsibility of the developer, this plugin doesn't provide any tool to help on this task.
